code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1886,6 +1886,10 @@ GLOBAL_LIST_EMPTY(mentor_races)
 				H.adjustStaminaLoss(damage * hit_percent * H.physiology.stamina_mod)
 		if(BRAIN)
 			H.adjustOrganLoss(ORGAN_SLOT_BRAIN, damage * hit_percent * H.physiology.brain_mod)
+	
+	if(H.buckled && istype(H.buckled, /obj/structure))//prevent buckling corpses to chairs to make indestructible projectile walls
+		var/obj/structure/sitter = H.buckled
+		sitter.take_damage(damage, damagetype)
```

Comment:
i would add a vis message to indicate that the chair is taking damage

Reply:
it plays a sound effect